website_crm_address/controllers/main.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    This module copyright (C) 2015 Therp BV (<http://therp.nl>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import http
+from openerp.addons.website_crm.controllers.main import contactus
+
+
+class ContactUs(contactus):
+    @http.route(['/crm/contactus'])
+    def contactus(self, **kwargs):
+        if 'zipcode' in kwargs:
+            kwargs['zip'] = kwargs['zipcode']
```

Comment:
`zip` is a python builtin, and if you do what you propose, you get the function reference as value